src/Tests/ObjectLifetimeTests/ObjectLifetimeTests.Lifted/ObjectLifetimeTests.Lifted/ObjectLifetimeTests.Lifted.csproj

```diff
@@ -0,0 +1,118 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+        <OutputType>WinExe</OutputType>
+        <TargetFramework>net5.0-windows10.0.19041.0</TargetFramework>
+        <TargetPlatformMinVersion>10.0.17763.0</TargetPlatformMinVersion>
+        <RootNamespace>ObjectLifetimeTests.Lifted</RootNamespace>
+        <ApplicationManifest>app.manifest</ApplicationManifest>
```

Comment:
Can we make these tests use the Windows SDK and WinUI projection we build in the cswinrt repo (the ones under the projection folder)

Reply:
Shashank and I chatted about how to do this.  I suggested renaming (via AssemblyName project prop) our test projection to Microsoft.WinUI.dll, possibly setting major version to something artificially high (e.g., 99), and possibly strong name signing.  That should match link requirements of MSTest and ensure that the later version overrides.